def.bzl

```diff
@@ -57,6 +74,12 @@ def _gazelle_runner_impl(ctx):
         args.extend(["-build_tags", ",".join(ctx.attr.build_tags)])
     args.extend([ctx.expand_location(arg, ctx.attr.data) for arg in ctx.attr.extra_args])
 
+    for key in ctx.attr.env:
+        if not _valid_env_variable_name:
```

Comment:
You aren't calling the function here, you are taking it's truth value (which is always `True`). If you were to call it, I think it would fail since in Starlark strings aren't iterable. You need to use `name.elems()` to get the chars.

Reply:
yeah should have gotten more coffee :P